packages/vite-plugin-angular/src/lib/angular-vite-plugin.ts

```diff
@@ -368,7 +368,9 @@ export function angular(options?: PluginOptions): Plugin[] {
           }
 
           if (
-            (id.endsWith('.analog') || id.endsWith('.agx')) &&
+            (id.endsWith('.analog') ||
+              id.endsWith('.agx') ||
```

Comment:
question: What exactly is `*.agx`? `agx` isn't set up in all the places where `ag` is set up now. Looks like an `mdx` equivalent with a markdown focus?

Reply:
Essentially yes, it allows using analog components as content files with front matter

Reply:
Ah, okay. I was under the impression that they were all just aliases of each other. Thanks!